File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/TaggedDriverBase.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id The ID of the Driver being tagged.
 */
data class TaggedDriverBase (

        @get:NotNull 
        @JsonProperty("id") val id: Long
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/TaggedDriverBase.scala

package org.openapitools.client.model


case class TaggedDriverBase (
    /* The ID of the Driver being tagged. */
    _id: Long
)
object TaggedDriverBase {
    def toStringBody(var_id: Object) =
        s"""
        | {
        | "id":$var_id
        | }
        """.stripMargin
}
